src/components/validators/project-name-available-validator.js

```diff
@@ -28,6 +31,10 @@
 
             return deferred.promise;
           };
+
+          scope.$watch("organizationId", function() {
+            ngModel.$validate();
```

Comment:
Thanks, for the heads up. I just wonder if there is a $validateAsync() method or if calling this will trigger the async validation (it's been a while since I dove deep into the validation pipeline).